pkg/rpc/client/policy.go

```diff
@@ -12,35 +12,53 @@ import (
 // GetMaxTransactionsPerBlock invokes `getMaxTransactionsPerBlock` method on a
 // native Policy contract.
 func (c *Client) GetMaxTransactionsPerBlock() (int64, error) {
-	return c.invokeNativePolicyMethod("getMaxTransactionsPerBlock")
+	if !c.initDone {
+		return 0, errNetworkNotInitialized
+	}
+	return c.invokeNativeGetMethod(c.cache.nativeHashes[nativenames.Policy], "getMaxTransactionsPerBlock")
 }
 
 // GetMaxBlockSize invokes `getMaxBlockSize` method on a native Policy contract.
 func (c *Client) GetMaxBlockSize() (int64, error) {
-	return c.invokeNativePolicyMethod("getMaxBlockSize")
+	if !c.initDone {
+		return 0, errNetworkNotInitialized
+	}
+	return c.invokeNativeGetMethod(c.cache.nativeHashes[nativenames.Policy], "getMaxBlockSize")
 }
 
 // GetFeePerByte invokes `getFeePerByte` method on a native Policy contract.
 func (c *Client) GetFeePerByte() (int64, error) {
-	return c.invokeNativePolicyMethod("getFeePerByte")
+	if !c.initDone {
+		return 0, errNetworkNotInitialized
```

Comment:
It's so repetetive that it could probably be done in another wrapper (deduplicating this).